package.json

```
@@ -11,10 +11,18 @@
     "mock",
     "URI"
   ],
-  "main": "index.js",
+  "main": "cjs/index.js",
+  "module": "mjs/index.js",
+  "exports": {
+    ".": {
+      "import": "./mjs/index.js",
+      "require": "./cjs/index.js"
+    }
```

Can we also add an export for the `package.json`? Many ESM packages with an export map include a `package.json` export so that users can do `import('@cloudfour/simple-svg-placeholder/package.json')`, e.g. to get the version or something.

The syntax would look like this: https://github.com/preactjs/preact/blob/master/package.json#L89

```suggestion
    },
    "./package.json": "./package.json",
```